apps/src/aichat/views/AichatView.tsx

```diff
@@ -158,17 +158,17 @@ const AichatView: React.FunctionComponent = () => {
   const viewModeButtonsProps: SegmentedButtonsProps = {
     buttons: [
       {
-        label: 'Edit',
+        label: aichatI18n.editModeButtonText(),
         value: ViewMode.EDIT,
-        iconLeft: {iconName: 'wrench', iconStyle: 'solid', title: 'Edit Mode'},
+        iconLeft: {iconName: 'wrench', iconStyle: 'solid', title: 'Edit Mode'}, // if this is only used by a screen reader, do we translate it?
```

Comment:
When we were testing Music Lab panel levels, the browser-based TTS just read the English text in an accented voice unless the text was translated, IIRC (cc @breville @sanchitmalhotra126). I suspect the answer is that we _should_ translate here, but I have no idea how screen readers in other languages work. Maybe a question for #accessibility?

Reply:
Asked here https://codedotorg.slack.com/archives/C9MB4CL07/p1730768413417189

Reply:
Answer: Yes, we should translate the title